hw2/styles.css

```diff
@@ -0,0 +1,53 @@
+/*02-search*/
```

Comment:
Adding more than one styles file for each assignment could be a nice touch as it can reduce conflicts between similar elements names--but it could be avoided.

Reply:
Although having one for each one is an amazing idea if we are looking to work more on it, I feel for this assignment, as each of it does not contain a lot of content in the style, I believe it is not needed to add more than one styles file.